app/assets/stylesheets/application.scss

```diff
@@ -26,15 +34,74 @@ body {
   text-align: center;
 }
 
-.welcome {
-  background: rgba(255, 255, 255, 0.5);
-  padding: 10px;
-}
-
-.footer {
-  background-color: #333333;
+footer {
+  background-color: $darkslategrey;
   bottom: 0;
   color: #ffffff;
-  position: fixed;
-  width: 100%;
+}
+
+.top-bar ul li {
+  background: $darkslategrey;
+}
+
+.top-bar a {
+  font-size: 16px;
+}
+
+.top-bar-section img {
+  background: inherit;
+  float: right;
+  margin-top: 7px;
+  margin-right: 0;
+  max-height: 3.5em;
+  width: 3.5em;
+}
+// Footer
+html,
+body {
+	height: 100%;
+}
+
+.page-wrap {
+	margin-bottom: -142px;
+  min-height: 100%;
+  /* equal to footer height */
+}
+
+.page-wrap:after {
+  content: "";
+  display: block;
+}
+
+.site-footer,
+.page-wrap:after {
+  height: 142px;
+}
+
+.site-footer {
+  background: $darkslategrey;
+}
+
+#icon {
+  width: 100px;
+}
+
+.large-4 {
+  border: 5px solid $mediumaquamarine;
+  background:rgba(255, 255, 255, 0.5);
+}
+
+.panel {
+  margin-top: 5px;
```

Comment:
Properties should be ordered border-color, border-left-width, border-right-width, margin-top